File: src/main/java/com/sedmelluq/discord/lavaplayer/filter/UniversalPcmAudioFilter.java
package com.sedmelluq.discord.lavaplayer.filter;

/**
 * A PCM filter which must be able to correctly process all representations of PCM input data, as specified by the
 * methods of the interfaces it extends.
 */
public interface UniversalPcmAudioFilter extends ShortPcmAudioFilter, SplitShortPcmAudioFilter, FloatPcmAudioFilter {
}

File: main/src/main/java/com/sedmelluq/discord/lavaplayer/filter/UniversalPcmAudioFilter.kt
package com.sedmelluq.discord.lavaplayer.filter

/**
 * A PCM filter which must be able to correctly process all representations of PCM input data, as specified by the
 * methods of the interfaces it extends.
 */
interface UniversalPcmAudioFilter : ShortPcmAudioFilter, SplitShortPcmAudioFilter, FloatPcmAudioFilter
